### ISSUE TITLE
[Question] 单卡训练正常，多卡训练loss == 0

### ISSUE THREAD

**Issue by Camellia-hz** [haotian-liu/LLaVA#1661]
I did pretrain and finetune based on llava-v1.5-7b, replacing the encoder with dinov2, and then I'm going to do lora-finetune on my own dataset, but when I train with a single A100 * 40G, the loss is normal, but once I train with multiple cards, the losshas been 0.

**Comment by Camellia-hz** [comment]
@simonw

**Comment by liziniu** [comment]
I have the same issue

**Comment by terry-for-github** [comment]
same issue too.

**Comment by zc-zhao** [comment]
same issue too

**Comment by bofei5675** [comment]
Address this issue by setting `scripts/zero2.json` or `scripts/zero3.json`

```json
{
    "overlap_comm": false
}
```

**Comment by user074** [comment]
> Address this issue by setting `scripts/zero2.json` or `scripts/zero3.json`
> 
> ```json
> {
>     "overlap_comm": false
> }
> ```

Thanks! Solved my issue

**Comment by Camellia-hz** [comment]
> Address this issue by setting `scripts/zero2.json` or `scripts/zero3.json`
> 
> ```json
> {
>     "overlap_comm": false
> }
> ```

Thanks, solved my problem!
